makefiles/variables.mk

```diff
@@ -14,7 +14,13 @@ MUSL_HOME?=/usr/local/musl
 # Tax computation configuration
 ##################################################
 
-ifeq ($(YEAR), 2022)
+ifeq ($(YEAR), 2023)
+    $(warning WARNING: the source M files and fuzzer tests have not yet been published for year: 2023)
+	SOURCE_FILES=$(call source_dir,$(ROOT_DIR)/ir-calcul/M_SVN/$(YEAR)/)
```

Comment:
My understanding was that viewing two different years as two completely different applications implied that having a "default" made no sense for our cases (it would be a dangling `cibles.m` that couldn't be linked to any other M files). I'm happy to keep 2023 as a draft PR for testing purposes only as long as there's no officially released M code for 2023